pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -466,10 +466,10 @@
     "owner": "heroku",
     "provider-source-address": "registry.terraform.io/heroku/heroku",
     "repo": "terraform-provider-heroku",
-    "rev": "v4.9.0",
-    "sha256": "0h61wldnjmmqfhjxvjf6jh0h6v22zx7rmff48pw1dyir19di7mdr",
-    "vendorSha256": "183j20my61mqhch87cn455l938jqx4ssbnv8yl416i6bxgcpy6f9",
-    "version": "4.9.0"
+    "rev": "v5.0.0-beta.1",
+    "sha256": "1dskbwa10dmj5fdw0wplby6hhcvxri68jlg34966mqx8pas3zsxy",
+    "vendorSha256": "13f7841i14b5n5iabqky7694mbqg95f0cvaygapczki5lf2j7fqy",
+    "version": "5.0.0-beta.1"
```

Comment:
@zimbatm Do you think we should start filtering pre-release versions? I had hoped we wouldn't need to bother with that but it seems like a lot of upstreams don't realise that the registry is picking up new tags as latest even if they are pre-release on github.